xml/2018-10-01.xml

```diff
@@ -0,0 +1,232 @@
+    technical skills or manpower.
+
+    **Operations -** DoCSoc has a central [Facebook group](https://www.facebook.com/groups/1509850436007222/)
+    for when we need volunteers. If you're thinking of volunteering, join the group!
+    Events we will need help for over the coming months include:
+
+    - Google DevFest London - We're hosting Europe's largest free developer conference
+    with over 1000 attendees on Saturday 1st December. We're looking for volunteers
+    to help with organising both before and during the event.
+
+    - IC Hack - With 300 attendees, our yearly hackathon won't be possible without
+    our volunteers. If you aren't looking to compete in the hackathon, lending a
+    helping hand is also a great way to participate in the hackathon!
+
+    **Software Foundation -** We're reviving our open source initiative, the
+    [Software Foundation](https://github.com/docsocsf) and are in need of
+    volunteers and contributors. This year, we're planning lots of projects,
+    from beginner-friendly to more advanced. If you'd like to get involved,
+    either as a project owner or only to suggest ideas, please let us know!
+  </text>
+
+  <hr />
+
+  <text>Now for the upcoming events:</text>
+
+  <heading>Freshers' Fair</heading>
+  <text>
+    - **When:** **TOMORROW!** 11:00 - 16:00, Tuesday 2nd October 2018
+    - **Where:** Main Entrance (ICME) Stall 5
+
+    Come and visit us at our stall at the Freshers' Fair to learn more about us,
+    chat with the committee and compete in a (yet to be announced) competition to
+    win some prizes! We'll be there all day so make sure to check out all of the
+    other fantastic opportunities as well. You can find us by consulting the official
+    map.
+  </text>
+
+  <heading>Mums &amp; Dads Lunch</heading>
+  <text>
+    - **When:** 12:30 - 14:00, Monday 8th October 2018
+    - **Where:** HXLY 341/342
+
+    As a fresher, you will have already heard of our Mums and Dads (MaD) programme.
+    If you don't have parents assigned yet or haven't heard of this programme, please
+    reach out to your departmental representatives above.
+
+    We'll be kicking off the Mums and Dads programme with an opportunity for you
+    to meet your MaD parents and have lunch with them! We will be offering free
+    food and drinks, as well as issuing a number of challenges. You might even win
+    a prize!
+
+    Mums and Dads parent? Be sure to turn up to this event to meet your children!
+  </text>
+
+  <heading>Sponsors' Exhibition</heading>
+  <text>
+    - **When:** 17:00 - 20:00, Tuesday 9th October 2018
+    - **Where:** QTR
+
+    Looking for an internship? Want to find out more about industry and chat
+    with our sponsors? Come join us in the Great Hall to have a chat with
+    companies ranging from Facebook to Palantir and Netcraft to Microsoft in our
+    premier autumn event. Bring lots of questions and be prepared to go home with
+    a lot of swag!
+  </text>
```

Comment:
Looking for an internship, placement or graduate role? Want to find out more about industry and chat with out sponsors? Come join us in Queen's Tower Room (Sherfield Building) for our main industrial event of the year, to have a chat with companies ranging from Facebook to Palantir and from Netcraft to Microsoft. Bring lots of questions and be prepared to go home with a lot of swag!